nbs/10_compute_correlations/05_gtex_v8/05-gtex-pearson.ipynb

```diff
@@ -0,0 +1,1037 @@
+       "gene_ens_id        ENSG00000198938.2  ENSG00000198886.2  ENSG00000198899.2\n",
+       "gene_ens_id                                                               \n",
+       "ENSG00000198938.2           1.000000           0.519992           0.639912\n",
+       "ENSG00000198886.2           0.519992           1.000000           0.833569\n",
+       "ENSG00000198899.2           0.639912           0.833569           1.000000"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "_tmp = CORRELATION_METHOD(test_data.iloc[:3])\n",
+    "\n",
+    "display(_tmp.shape)\n",
+    "display(_tmp)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "d32281d8-f640-4f87-a1d5-aefa0757e9c3",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-09-01T16:47:39.138743Z",
+     "iopub.status.busy": "2021-09-01T16:47:39.138211Z",
+     "iopub.status.idle": "2021-09-01T16:47:43.686416Z",
+     "shell.execute_reply": "2021-09-01T16:47:43.685938Z"
+    },
+    "papermill": {
+     "duration": 4.568251,
+     "end_time": "2021-09-01T16:47:43.686520",
+     "exception": false,
+     "start_time": "2021-09-01T16:47:39.118269",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "56.2 ms ± 928 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)\n"
+     ]
+    }
+   ],
+   "source": [
+    "%timeit CORRELATION_METHOD(test_data.iloc[:PERFORMANCE_TEST_N_TOP_GENES])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc2f7184-c0da-438c-94f5-002912793636",
+   "metadata": {
+    "papermill": {
+     "duration": 0.015392,
+     "end_time": "2021-09-01T16:47:43.716801",
+     "exception": false,
+     "start_time": "2021-09-01T16:47:43.701409",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "## Run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "1237ae45-d4fc-4074-bfc7-a1de75aeea48",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-09-01T16:47:43.752227Z",
+     "iopub.status.busy": "2021-09-01T16:47:43.751610Z",
+     "iopub.status.idle": "2021-09-01T16:50:25.222182Z",
+     "shell.execute_reply": "2021-09-01T16:50:25.221737Z"
+    },
+    "papermill": {
+     "duration": 161.490684,
+     "end_time": "2021-09-01T16:50:25.222303",
+     "exception": false,
+     "start_time": "2021-09-01T16:47:43.731619",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "gtex_v8_data_whole_blood-var_raw: 100%|█████████████████████████████| 54/54 [02:41<00:00,  2.99s/it]\n"
+     ]
+    }
+   ],
+   "source": [
+    "pbar = tqdm(input_files, ncols=100)\n",
+    "\n",
+    "for tissue_data_file in pbar:\n",
+    "    pbar.set_description(tissue_data_file.stem)\n",
```

Comment:
I didn't realize you could do this with TQDM, that's really cool

Reply:
Ah, set a description? Yes, `tqdm` is great.